values.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "airflow": {
+      "type": "object"
+    },
+    "authSidecar": {
+      "type": "object"
+    },
+    "extraObjects": {
+      "type": "array"
+    },
+    "helmValuesRevisionId": {
+      "type": "integer"
+    },
+    "ingress": {
+      "type": "object"
+    },
+    "platform": {
+      "type": "object"
+    },
+    "sccEnabled": {
+      "type": "boolean"
+    },
+    "workers": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "affinity": {
+          "type": "object"
+        },
+        "autoscaling": {
```

Comment:
`autoscaling` is the only thing that should be here. The others, if they exist, _should_ be an error as they should be under `airflow.workers` instead.

https://github.com/astronomer/airflow-chart/blob/d0c176ddaf42e1bd53e48c1c063057394aee464d/values.yaml#L340-L347